cogs/commands.py

```diff
@@ -370,7 +360,7 @@ async def join(self, ctx, *arg):
 
         # Display help if given no argument
         if not name:
-            return await ctx.send(ctx.command.help)
+            raise BadArgs("", ctx.command)
```

Comment:
By leaving the string blank, it will only show help. You could also use None or any other falsy Object. I think that a blank string looks best.